dockerfiles/steps/step-bake-web.bash

```diff
@@ -0,0 +1 @@
+source $PWD/../dockerfiles/steps/bake-util.bash web
```

Comment:
I am not sure why, but all builds are failing in concourse when this step starts. The log is not particularly useful, it only shows that the build failed on `source $step_file`:

> \+ [docker-entrypoint.sh:330] for input_dir in $input_dirs
> \++ [docker-entrypoint.sh:331] echo ''\''IO_RESOURCES'\'''
> \++ [docker-entrypoint.sh:331] tr -d \'
> \+ [docker-entrypoint.sh:331] check_input_dir IO_RESOURCES
> \+ [docker-entrypoint.sh:116] [[ -n IO_RESOURCES ]]
> \+ [docker-entrypoint.sh:117] pointer=IO_RESOURCES
> \+ [docker-entrypoint.sh:118] dir_name=resources
> \+ [docker-entrypoint.sh:119] [[ -n resources ]]
> \+ [docker-entrypoint.sh:120] [[ -d resources ]]
> \+ [docker-entrypoint.sh:333] for output_dir in $output_dirs

(end of log)

I did a `fly hijack` on the container and found the problem. 

This is what is contained within the directory this command runs inside of:
```
root@<long-uuid>:/tmp/build/4e43a059 ls 
assembled  baked  book  common-log  fetched  resources
```
and bake-utils.bash is in `/dockerfiles/steps/bake-util.bash`